test/testother.cpp

```diff
@@ -3089,46 +3090,46 @@ class TestOther : public TestFixture {
               "    x.dostuff();\n"
               "    const U& y = (const U&)(x);\n"
               "}");
-        ASSERT_EQUALS("[test.cpp:4]: (style) C-style reference casting\n"
-                      "[test.cpp:2]: (style) Parameter 'x' can be declared as reference to const\n",
+        ASSERT_EQUALS("[test.cpp:4:19]: (style) C-style reference casting [cstyleCast]\n"
+                      "[test.cpp:2:11]: (style) Parameter 'x' can be declared as reference to const [constParameterReference]\n",
                       errout_str());
         check("struct T : public U { void dostuff() const {}};\n"
               "void a(T& x) {\n"
               "    x.dostuff();\n"
               "    U& y = (U&)(x);\n"
               "    y.mutate();\n" // to avoid warnings that y can be const
               "}");
-        ASSERT_EQUALS("[test.cpp:4]: (style) C-style reference casting\n", errout_str());
+        ASSERT_EQUALS("[test.cpp:4:12]: (style) C-style reference casting [cstyleCast]\n", errout_str());
         check("struct T : public U { void dostuff() const {}};\n"
               "void a(T& x) {\n"
               "    x.dostuff();\n"
               "    const U& y = (typename const U&)(x);\n"
               "}");
-        ASSERT_EQUALS("[test.cpp:4]: (style) C-style reference casting\n"
-                      "[test.cpp:2]: (style) Parameter 'x' can be declared as reference to const\n",
+        ASSERT_EQUALS("[test.cpp:4:0]: (style) C-style reference casting [cstyleCast]\n"
```

Comment:
This lacks the column. I filed https://trac.cppcheck.net/ticket/13766 about that.